src/styles/price.css

```diff
@@ -0,0 +1,32 @@
+.product-card__price {
+  justify-content: space-between;
+  height: 18px;
+}
+
+.product-card__price-text {
+  position: absolute;
```

Comment:
Don't use absolute positioning here. It is unnecessary. Use standard flow of HTML elements and margins for setting distance between them.